components/logins/src/db.rs

```diff
@@ -30,22 +30,26 @@ use url::{Host, Url};
 
 #[derive(Serialize, Deserialize, PartialEq, Debug, Clone, Default)]
 pub struct MigrationPhaseMetrics {
-    num_processed: u64,
-    num_succeeded: u64,
-    num_failed: u64,
-    total_duration: u128,
-    errors: Vec<String>,
+    pub(crate) num_processed: u64,
+    pub(crate) num_succeeded: u64,
+    pub(crate) num_failed: u64,
+    // Once https://github.com/mozilla/uniffi-rs/pull/434 lands we might
+    // want to make this a `Duration`?
```

Comment:
👍🏻 thanks for leaving these breadcrumb comments, they're helpful as we co-evolve the tool and its consumers.